src/views/Dashboard.vue

```diff
@@ -62,10 +62,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               <v-icon size="1.6em">{{ $options.icons.table }}</v-icon>
             </template>
             <v-list-item-title class="text-h6 font-weight-light">
-              Workflows Table
+              Workflow Table
```

Comment:
The dashboard could use a better link name here, to indicate what distinguishes this "Workflow(s) Table" from the workflows listed in the side bar.  It's basically workflow contact information: owner, host, post (it should have PID too) plus status (running, stopped, paused). What can we call this?  *Workflow Status*, or *Workflow Details*, or *Workflow Locations*, ...?

Reply:
> to indicate what distinguishes this "Workflow(s) Table" from the workflows listed in the side bar

The thing that distinguishes it, is being a table!

Reply:
I'll kill this diff.